internal/server/proto/server.proto

```diff
@@ -2387,6 +2387,9 @@ message StatusReport {
     // Resource id
     string id = 4;
   }
+
+  // the time when this report was generated
+  google.protobuf.Timestamp generated_time = 10;
```

Comment:
We should leave a note on line 2370 that these variables (health, resources_health, and generated_time) are all copied from the plugin generated raw SDK status report and are a convenience for the UI and CLI. Also move this line to under `resources_health` so that the Health message is the last thing here (that's mostly a style thing)